picard/ui/searchdialog/album.py

```diff
@@ -135,36 +139,43 @@ def invalidate(self):
         if self.widget:
             self.widget = None
 
+    def __repr__(self):
+        return "CoverCell(%r, %r, %r, %r, %r, on_show=%r)" % (self.table, self.row, self.column, self.mbid, self.size, self.on_show)
```

Comment:
As you have this currently it's fine (perhaps even better), but I probably would have done this as an f-string as:

```python
return f"CoverCell({self.table!r}, {self.row!r}, {self.column!r}, {self.mbid!r}, {self.size!r}, on_show={self.on_show!r})"
```

I would have probably used the f-string because I try to use them by default unless there is a reason not to (such as in log entries where the resulting string might not be used).  Is there any advantage or disadvantage to using one or the other format in this case, so that I can make better usage decisions in the future?

Reply:
Maybe I just answered my own question after reviewing https://peps.python.org/pep-0498/#s-r-and-a-are-redundant.

Reply:
I went for a slightly different approach in https://github.com/metabrainz/picard/pull/2658/commits/6a1f4f5daf5a6db78b13af8357690dd43898e71e